app/scripts/services/listSubjectsFactory.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Created by kmuzytc on 23.03.2015.
+ */
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name admissionSystemApp.controller:MainCtrl
+ * @description
+ * # MainCtrl
+ * Controller of the admissionSystemApp
+ */
+angular.module('admissionSystemApp')
+
+//http://176.36.11.25/api-lnu/enrolments/subjects
+//http://104.236.29.16:8080/is-lnu-rest-api/api/enrolments/subjects
+  .factory('getSubjects', ['$http', '$q', function($http, $q) {
```

Comment:
1. Factory and file should have the same name.
2. Factory is an entity, so it must be named as noun.